keyboards/archetype/minervalx/readme.md

```diff
@@ -0,0 +1,24 @@
+# Minerva LX
+
+![Minerva LX](https://archetypemade.com/minerva-is-here/)
```

Comment:
This isn't an image link...

Reply:
I added the image link in the commit b470462